corehq/apps/userreports/indicators/specs.py

```diff
@@ -87,10 +90,3 @@ class ChoiceListIndicatorSpec(PropertyReferenceIndicatorSpecBase):
 
     def get_operator(self):
         return IN_MULTISELECT if self.select_style == 'multiple' else EQUAL
-
-
-def _transform_from_datatype(datatype):
-    return {
-        'date': transform_date,
-        'integer': transform_int,
-    }.get(datatype)
```

Comment:
@czue 
So we've slightly changed the behavior or `RawIndicatorSpec` and `ExpressionIndicatorSpec`. These previously required a datatype, but the "string" datatype was actually not converting the datatype at all because `_transform_from_datatype` was returning `None`. But, the new `transform_from_datatype` function does convert values to unicode when a datatype of "string" is returned. I believe case properties and question answers are stored as strings in couch, but empty properties are `null`/`None`. So, now "string" datatype converts `None` to `"None"`, and hence the test failures. I'm thinking about what the ideal behavior would be...